index.js

```diff
@@ -30,15 +31,13 @@ connectDB()
         logger.error(`Database connection failed: ${err.message}`);
     });
 
-
 // user Route
-
 app.use(responseInterceptor);
-
-app.use('/api/v1/user', userRouter)
-
+app.use('/api/v1/user', userRouter);
+app.use('/api/v1/user', visitorRouter);
```

Comment:
create common routes file and wrap all there

Reply:
done